DOL.WHD.Section14c.Api/packages.config

```diff
@@ -32,6 +32,9 @@
   <package id="Microsoft.Owin.Security.Twitter" version="3.0.1" targetFramework="net461" />
   <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net461" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net461" />
+  <package id="NLog" version="4.4.2" targetFramework="net452" />
+  <package id="NLog.Web" version="4.5.0" targetFramework="net452" />
+  <package id="Npgsql" version="3.2.5" targetFramework="net452" />
```

Comment:
Do we need Postgres driver?

Reply:
I am not sure why the NPSQL is adding to the project. It's already exists in the current solution. I will remove them.